stellar-auth/pkg/auth/mfa_manager.go

```diff
@@ -147,6 +149,31 @@ func (m *defaultMFAManager) ForgetDevice(ctx context.Context, deviceID, userID s
 	return nil
 }
 
+func (m *defaultMFAManager) GetAuthUserID(ctx context.Context, deviceID string) (string, error) {
```

Comment:
👀 `auth` is just a prefix used in the database for auth-related tables. The correct naming here according with the rest of the code conventions would be `GetUserID` or `GetUserIDByDevideID`.